backpack-common/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -145,6 +145,17 @@ data class CellInfo(
     }
 }
 
+/**
+ * Describes the label of the cell
+ */
+sealed class CellLabel {
+    data class Text(val text: String) : CellLabel()
+    data class Icon(
+        val resId: Int? = null,
```

Comment:
I think this could be non-nullable. Can't see a case where you'd create an `Icon` without a resource id?

Reply:
Good call, I'll update it. Thanks